### ISSUE TITLE
Change initialization of local variables in MPFA-O

### ISSUE THREAD

**Issue by bishtgautam** [TDycores-Project/TDycore#108]
Change
```
  DM             dm;
  TDy_mesh       *mesh;
  TDy_cell       *cells;

  dm       = tdy->dm;
  mesh     = tdy->mesh;
  cells    = &mesh->cells;
```

to 
```
  DM dm       = tdy->dm;
  TDy_mesh *mesh    = tdy->mesh;
  TDy_cell *cells    = &mesh->cells;
```

**Comment by jeff-cohere** [comment]
This is a good idea--it allows you to define local variables when you need them, and at the proper scope.

**Comment by bishtgautam** [comment]
Fixed in #124
